site/docs/v1/tech/apis/_webhook-search-request-parameters.adoc

```diff
@@ -0,0 +1,38 @@
+
+// parameter_prefix is either blank for parameters or "search." for body
+
+[.api]
+
+[field]#{parameter_prefix}description# [type]#[String]# [optional]#Optional#::
+The case-insensitive string to search for in the Webhook description or URL. This can contain wildcards using the asterisk character (`*`). If no wildcards are present, the search criteria will be interpreted as `pass:[*value*]`. If this field is omitted, no filtering will be applied to the `description` or `url` fields.
```

Comment:
Seems weird to not have a URL search field.

Reply:
Part of the problem here is how to present the search options in the UI. We'd prefer not to have two text fields on the front-end. There are a couple of other of these search APIs that use a single request field to search multiple database fields.

The other thing is deciding how to perform the search if `description` and `url` are separate fields on the request. The current code checks whether there is a match in description `OR` URL. If we were to have separate request fields, should the result still be `OR`'d? Should it be an `AND` where both have to match? Do we need another parameter to decide which logical operator is applied?

I think that the single request parameter that checks both fields is simpler to understand and use, and I don't think having a separate field adds much value.

Reply:
I think Dan is correct. This is one of the changes I made to the Search APIs prior to merging. 

The API is completely separate from how we choose to use the API in our admin UI. Think of them as separate applications. 

This means we cannot use one field `description` to search on another, this would be a surprising API behavior.